examples/nodejs/lambda-examples/topics-microservice/cdk.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
woof is this really what they are generating these days? the ones we use are usually much shorter, but if this is what they generated, it's fine

Reply:
This is what the CDK tool generated when I did `cdk init`. *shrug*